src/functions.php

```diff
@@ -2,6 +2,7 @@
 
 namespace React\Async;
 
+use Fiber;
```

Comment:
nit: Should be omitted (see above)?

Reply:
There was actually one line that didn't use `\Fiber`